packages/react-components/src/molecules/UsersList.tsx

```diff
@@ -38,7 +38,7 @@ const itemStyles = css({
 const userStyles = css({
   overflow: 'hidden',
   display: 'grid',
-  gridTemplateColumns: 'min-content 1fr',
+  gridTemplateColumns: 'min-content 1fr 0.5fr',
```

Comment:
```suggestion
  gridTemplateColumns: 'min-content 1fr min-content',
```
Why did you use a fraction, the first icon used min-content.